### ISSUE TITLE
House isocon iteration

### ISSUE THREAD

**Issue by jehlers** [cfpb/owning-a-home#3]
At the CFPB, we have three different levels of iconography and illustration that we use throughout our web and print experiences. Our middle tier, which we call isocons (they are icons drawn in isometric), are larger-scale icons used to illustrate values, products, topics, and processes. 

As we kick off this project, we need an isocon to represent the Owning a Home product. The graphic design team has struggled with how to best represent a home in isometric for quite a while, so it was important that we push through lots of iterations and finally get it right. 

Here is our current isocon library, so you can get a feel for the visual language we needed to design within:
![14df75f2-94de-11e3-8413-e6e1d6b7ac1e](https://f.cloud.github.com/assets/4267825/2295875/17002be2-a096-11e3-93da-6bbeb45bb1ec.png)

We started by grabbing the basic shapes from the bank isocon: 
![f2d75490-8847-11e3-8b11-223fa5c8006a](https://f.cloud.github.com/assets/4267825/2295876/1ac26bd2-a096-11e3-90e3-f9474eff8f52.png)

Then we broke down the elements of the house (windows, door, roofing, lot) and went to work iterating:
![c0158882-94dc-11e3-88dd-012c238b73e2](https://f.cloud.github.com/assets/4267825/2295878/201c4df0-a096-11e3-89fb-18ec6a3906f9.gif)

Ultimately, we settled on a simple house that felt consistent with the rest of our library in its level of detailing:
![043dc678-94dd-11e3-9b84-b41cb4ee47f6](https://f.cloud.github.com/assets/4267825/2295880/24b7d9ba-a096-11e3-9c74-210dacf5ca2a.png)

**Comment by mebates** [comment]
I don't see this house in the Isocon library on Drive. Who has the native file? Can it be added?

cc: @stephanieosan @ohsk

**Comment by stephanieosan** [comment]
I've got the native file, should I just add it to the library on the Drive myself?

**Comment by mebates** [comment]
yeah! update the library, date it, and throw the old one in the archive folder.
